tests/src/test/scala/cats/tests/KleisliTests.scala

```diff
@@ -93,6 +93,12 @@ class KleisliTests extends CatsSuite {
     checkAll("SemigroupK[Lambda[A => Kleisli[Option, A, A]]]", SerializableTests.serializable(kleisliSemigroupK))
   }
 
+  {
+    implicit val kleisliContravariant = Kleisli.kleisliContravariant[Option, Int]
```

Comment:
I've gone through and changed KleisliTests to not explicitly specify the implicits.